targets/nintendoswitch.json

```diff
@@ -9,9 +9,11 @@
   "libc": "picolibc",
   "gc": "conservative",
   "relocation-model": "pic",
+  "features": ["+a57","+strict-align","+crc","+crypto"],
   "cflags": [
-    "-target", "aarch64-none-linux-gnu",
+    "-target", "aarch64-unknown-none",
     "-mtune=cortex-a57",
+    "-mcpu=cortex-a57",
```

Comment:
The `-mcpu` flag should match the `cpu` field in the JSON file, see targets/esp32.json for example. I think that can replace the `features` field entirely.

Also, I think `-mtune` is unnecessary in this case, as it should tune for the processor specified in `-mcpu`.